src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1260,6 +1257,16 @@ forkEnvironment.setWorkingDir("/data/folder")
 forkEnvironment.setJavaHome("/usr/java/default")
 ```
 
+By default, the forked mode is configured then all workflow tasks are executed in a forked JVM separated from the ProActive Node's JVM.
```

Comment:
I would prefer that the general fork behavior of a task is explained (with a schema) is explained in the begining of this chapter, and then how the fork mode can be customized with fork environments scripts